libs/shared/contentful/README.md

```diff
@@ -0,0 +1,11 @@
+# shared-contentful
+
+This library was generated with [Nx](https://nx.dev).
+
+## Building
+
+Run `nx build shared-contentful` to build the library.
```

Comment:
This should be updated to include the type building instructions you mentioned to me, as well as when they should be done.